recipe/bld.bat

```diff
@@ -0,0 +1,22 @@
+if "%ARCH%"=="32" (
+   set MACHINE="IX86"
+   :: A different SDK is needed when build with VS 2017 and 2015
+   :: http://wiki.tcl.tk/54819
+   if "%VS_MAJOR%"=="14" (
+    echo "Switching SDK versions"
+    call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x86 10.0.15063.0
+   )
+) else (
+  set MACHINE="AMD64"
+  :: A different SDK is needed when build with VS 2017 and 2015
+  :: http://wiki.tcl.tk/54819
+  if "%VS_MAJOR%"=="14" (
+    echo "Switching SDK versions"
+    call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x64 10.0.15063.0
+  )
+)
+
+pushd win
+nmake -f makefile.vc libpath32="" CC=%CC% LD=link.exe TCL_VER=86 TCL_VERSION=8.6 DEST_DIR=%LIBRARY_PREFIX% MACHINE=%MACHINE% install
```

Comment:
isn't there an env var for `link.exe` on win?

Reply:
All I found is:
`set "LD=lld-link.exe"` in [clang-win-activation-feedstock/recipe/activate-clang_win-64.bat](https://github.com/AnacondaRecipes/clang-win-activation-feedstock/blob/15662eadd56ad0b0081f3e03b35d09f4b50799de/recipe/activate-clang_win-64.bat#L10)

Googling says:
```
By default, nmake uses the Microsoft Visual Studio linker (link.exe) for linking.
The LD variable is often set to link.exe by default.
```